tendrl/commons/utils/central_store/utils.py

```diff
@@ -92,6 +92,7 @@ def watch(obj, key):
 
     while True:
         try:
+            obj = obj.load()
```

Comment:
i feel doing load in all sync is very expensive one, so move this load logic under this line https://github.com/Tendrl/commons/pull/980/files#diff-cc2ad52a1d4152ad76a49b57cecfd59bR106 , because we are not using object anywhere except this block, so we can load object before on change call only.
and please verify once with that change